src/content/nft/index.md

```diff
@@ -77,16 +77,46 @@ This website has an alternative domain name powered by NFTs, **ethereum.eth**. O
 
 ## How do NFTs work? {#how-nfts-work}
 
-NFTs have some special properties:
-
-- Each token minted has a unique identifier.
+Thinking about this from a very high level, there are many blockchains that support NFTs but you will find majority under the Ethereum blockchain. There is a difference between NFT's and Ethereum tokens (ERC-20 or ETH) in that NFT's have extra information stored in them. This extra information is what really makes an NFT an NFT! NFT's are unique crypto tokens that are managed on a blockchain and the blockchain acts as the decentralized ledger that tracks the ownership and transaction history of each NFT. An NFT is created, or in the proper terms "minted" from digital objects that represent many things. For example, an NFT can consist of items such as:
```

Comment:
Couple thoughts here...
- First sentence doesn’t quite address the header it sits under. The section is “How do NFTs work?”… I would scrap that first sentence about multiple blockchains having them, etc. 

> There is a difference between NFT's and Ethereum tokens (ERC-20 or ETH) in that NFT's have extra information stored in them.

This reads a little funny for me, given that NFT's are also tokens. Maybe:
“NFTs are different from ERC-20 tokens, such as DAI or LINK, in that each individual token is completely unique and is not divisible.”

Other random thought while reading this:
- NFTs give the ability to assign/claim ownership to any unique piece of digital data

Reply:
Working on this now :)